### ISSUE TITLE
Possible issue with a checkout of https://github.com/httpie/cli/

### ISSUE THREAD

**Issue by pombredanne** [aboutcode-org/python-inspector#193]
As reported privately, running Python inspector on a checkout of https://github.com/httpie/cli/ may be problematic. More details to follow.

**Comment by sschuberth** [comment]
Yes, that's an issue we saw with [ORT](https://github.com/oss-review-toolkit/ort) (which uses `python-inspector` for Python projects) as well when taking part at the [SBOM Harmonization Plugfest](https://resources.sei.cmu.edu/news-events/events/sbom/). In short, no dependencies are reported at all. Which at a first glance *seems* to be correct as their [`setup.py` is basically empty](https://github.com/httpie/cli/blob/master/setup.py). But in exchange the [metadata is defined as part of `setup.cfg`](https://github.com/httpie/cli/blob/master/setup.cfg).
